### PR TITLE
STY: improve QC of unprocessed data

### PR DESCRIPTION
**Pull request by celprov**
enh: add QC criteria inspired from the Teves 2023 paper
fix: rename qaqc-criteria.md to qaqc-criteria-unprocessed.md

### CODE REVIEW

**Comment by oesteban** [comment]
@celprov - what is the status of this?

**Comment by celprov** [comment]
I revised the text of the QC on unprocessed data. I don't remember why I left it, in work in progress. 
I will have a look at this PR, #385, and work on separating QA vs exclusion criteria on fMRIPrep report in one block soonish.

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]: This is presented as some sort of feature, but it is actually a bug and is unrelated to the warning:
```suggestion
```

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
    !!! tip "Do not hesitate to jump back and forth through sections while screening the visual report"
```

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]: Semantic line breaks
```suggestion
- [ ] Verify that no errors are reported in subsection "Errors" of section "About".
    If there are, follow the procedure described in "*MRIQC* failed to produce all the expected visual reports".
```

Comment (oesteban) [reply docs/data-management/mriqc.md]: Also, it would be nice to add an anchor right before or inside the admonition block and convert that `"*MRIQC* failed to produce all the expected visual reports"` into a link.

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
    Except if the image is [extremely distorted](qaqc-criteria-unprocessed.md#task-fmri-exclusion-criteria), we are not excluding QCT scans because we will leverage those images to evaluate the quality of fMRI scans and derived constructs throughout the whole analysis pipeline.
    It is important to screen and rate the QCT runs as they may serve as a proxy for the quality of the other fMRI tasks in a session and display issues that might be affecting the RSfMRI and BHT.
```

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
- [ ] Visualize all echo-wise visualizations of the base report
- [ ] Inspect the background view and search for [artifacts](qaqc-criteria-unprocessed.md#view-of-the-background-of-the-voxel-wise-average-of-the-bold-timeseries) in the "Extended echo-wise reports" section.
```

Please describe what we are looking for in the echo-wise visualizations.

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
- [ ] Search for [artifacts](qaqc-criteria-unprocessed.md#average-signal-through-time) in the average BOLD mosaic visualization.
```

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
```

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
    - [ ] Search for [artifacts](qaqc-criteria-unprocessed.md#carpetplot-and-nuisance-signals) in the carpetplot visualization.
```

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]: Is this block the same of above? Can we merge the three of them into a single checklist that is applied to each task?

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]: ditto

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
- [ ] Visualize the IQM distributions and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#group-report-1).
```

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md]:
```suggestion
- [ ] Visualize the IQM distributions and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#group-report).
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
As such, verifying that the subjects were attempting to perform the instructed task (i.e., not sleeping and not responding randomly) is important. 
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
Because the one participant in Cohort I is the principal investigator, we assume that he is following all the instructions and reporting any issues as they occur.
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
Nonetheless, we will verify that the participant did not inadvertently fall asleep during fMRI runs.
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
- [ ] Exclude the BHT and QCT fMRI runs displaying extreme distortions of the image, reconstruction failures, electromagnetic spikes, ghost artifacts overlapping cortical gray matter, or extreme noise levels.
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]: If the image has problems in the header, we need to check the conversion to NIfTI instead.

```suggestion
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
    We acquired field maps in every session to address susceptibility distortions within the preprocessing.
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
    If signal dropouts in particular sessions are especially acute or widespread as compared to other sessions, double-check the visual report for other issues.
```

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]: Why these empty lines between list items? Please remove.

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md:153]: Please add an empty line after a header.

File: docs/data-management/qaqc-criteria-unprocessed.md
Comment (oesteban) [inline docs/data-management/qaqc-criteria-unprocessed.md]:
```suggestion
- [ ] Verify that smoothness estimates (FWHM) are roughly consistent across all sessions since they were all acquired with the same protocol.
    If this is not the case, re-examine the individual visual report of sessions with outlying (both low or high) FWHM.
```

**Review by oesteban — COMMENTED** [review]
Looking good! Left a few comments.

File: docs/data-management/mriqc.md
Comment (oesteban) [inline docs/data-management/mriqc.md:43]: Sweet :)

**Comment by oesteban** [comment]
Working on this, please do not merge yet :)

**Review by oesteban — APPROVED** [review]
I have made a commit --- LGTM!

**Pull request merged**
